pkg/storage/accessibility_request.go

```diff
@@ -0,0 +1,99 @@
+package storage
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/graph/model"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+// CreateAccessibilityRequest adds a new accessibility request in the database
+func (s *Store) CreateAccessibilityRequest(ctx context.Context, request *model.AccessibilityRequest) (*model.AccessibilityRequest, error) {
+	if request.ID == uuid.Nil {
+		request.ID = uuid.New()
+	}
+	createAt := s.clock.Now()
+	if request.CreatedAt == nil {
+		request.CreatedAt = &createAt
+	}
+	if request.UpdatedAt == nil {
+		request.UpdatedAt = &createAt
+	}
+	const createRequestSQL = `
+		INSERT INTO accessibility_request (
+			id,
+			name,
+			created_at,
+			updated_at
+		)
+		VALUES (
+			:id,
+			:name,
+		    :created_at,
+			:updated_at
+		)`
+	_, err := s.db.NamedExec(
+		createRequestSQL,
+		request,
+	)
+	if err != nil {
+		appcontext.ZLogger(ctx).Error(
+			fmt.Sprintf("Failed to create accessibility request with error %s", err),
```

Comment:
Instead of burying the `err` in the `Sprintf`, just use the zap-provided `error` type wrapper something like `...Error("failed to ...", zap.Err(err))`